src/pymatgen/analysis/local_env.py

```diff
@@ -1677,9 +1679,6 @@ def get_bonded_structure(self, structure: Structure, decorate: bool = False) ->
         Returns:
             MoleculeGraph: object from pymatgen.analysis.graphs
         """
-        # requires optional dependency which is why it's not a top-level import
-        from pymatgen.analysis.graphs import MoleculeGraph
```

Comment:
This is already eagerly imported (at runtime): https://github.com/materialsproject/pymatgen/blob/bd9fba9ec62437b5b62fbd0b2c2c723216cc5a2c/src/pymatgen/analysis/local_env.py#L27